trex-operator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/operator-framework/ansible-operator:v1.10.0
+FROM quay.io/operator-framework/ansible-operator:v1.33.0
 
 MAINTAINER telcoci@redhat.com
 
```

Comment:
- In LABEL, version must be updated to 0.2.17, and change valiation for validation (typo)
- These lines are not probably needed, they were removed in other operators already refactored:

```
USER root
RUN yum -y update-minimal --setopt=tsflags=nodocs \
        --security --sec-severity=Important --sec-severity=Critical
USER ansible
```

Reply:
Thanks, Done